components/edit-collective/sections/FiscalHosting.tsx

```diff
@@ -226,7 +184,7 @@ const FiscalHosting = ({ collective, account }) => {
   return (
     <div className="mb-10 flex w-full flex-col gap-4">
       <p>
-        <FormattedMessage defaultMessage="Manage the platform functionalities for your organisation" id="lVkMQs" />
+        <FormattedMessage defaultMessage="Manage the platform functionalities for your organization" id="lVkMQs" />
```

Comment:
Good catch, a French could have made this typo :)

Good if we can keep an auto-generated ID though:

```suggestion
        <FormattedMessage defaultMessage="Manage the platform functionalities for your organization" id="CETJRY" />
```